### PR TITLE
Update readme: PRs for translations not allowed

### LINKED ISSUE #12: Update README to point contributors to Pontoon
We should update the README to explain that changes through pull requests won't be accepted (with exceptions for specific locales), and that contributors should go to Pontoon.

Also, no point in mentioning l10n-central at this point, while the rest should probably stay.

### PR DESCRIPTION
**Pull request by bcolsson**
Fixes #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+If you’d like to submit translations or suggest improvements to existing
+strings please visit us on [Pontoon](https://pontoon.mozilla.org/projects/firefox/).
+Changes to translations will not be accepted by pull request
+(with exceptions for specific locales).
+
+## String updates
+
+This repo contains only localized files;
+source strings are located in the 
+[firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source/?tab=readme-ov-file)
```
Comment (flodolo) [inline README.md:14]:
```suggestion
[firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source/)
```

What's the purpose of the extra parameter?

**Review by flodolo — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eemeli) [reply README.md:14]: The base repo view can have additional tabs, such as: https://github.com/mozilla-l10n/firefox-l10n?tab=MPL-2.0-1-ov-file

The readme is the default, and unnecessary here.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
 # Firefox Localizations
 
-This is a monorepo replacement for the [l10n-central](https://hg.mozilla.org/l10n-central/) repositories.
+This repository contains the localized messages for all Firefox locales.
 
-While we're transitioning from hg to git,
+If you’d like to submit translations or suggest improvements to existing
+strings please visit us on [Pontoon](https://pontoon.mozilla.org/projects/firefox/).
+Changes to translations will not be accepted by pull request
+(with exceptions for specific locales).
```
[the remark this answers is not in the document]
Comment (bcolsson) [reply README.md]: Added specifics here - @flodolo let me know if it's too specific:
```
Changes to translations will not be accepted by pull request
with exceptions for specific locales:
- `it`: Translations committed to repo directly by @flodolo as part of source string localizability review process.
- `ja` and  `ja-JP-mac`: Translations are merged via PR due to legacy constraints.
```

[the remark this answers is not in the document]
Comment (flodolo) [reply README.md]: Looks accurate to me.

**Pull request merged**